### PR TITLE
WR-364: Theme layouts restructure

### PR DESCRIPTION
**Pull request by zetagraph**
Tickets: 

https://hive.aved.gov.bc.ca/jira/browse/WR-364

### CODE REVIEW

**Comment by zetagraph** [comment]
@Mirzero @infojunkie

There is quite a bit of restructuring here. Trying to remove the reliance on Drupal Blocks and Views in content and template rendering. It will give us a lot more flexibility in terms of markup and structure in various content types/nodes. 

Dave, I've set up a Career Profiles node template and can guide you through the setup if needed. But basically the `node--page--full.html.twig` can be viewed as an example. 

It might be a good idea to check this branch out and test things locally. I am removing a bunch of blocks and unneeded templates. Will also remove Hero and Title Views once we confirm that all is in order.

There are a few more updates that will be needed for news and success stories. Work in progress.

**Comment by infojunkie** [comment]
At first glance, this looks good. The lines to replace the hero views with simple if conditions are simple, as expected. I will leave it to @Mirzero to test the actual changes locally.

File: src/web/themes/custom/workbc/templates/layout/page--4xx.html.twig [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{#
```
Comment (Mirzero) [inline src/web/themes/custom/workbc/templates/layout/page--4xx.html.twig:1]: I'm unclear what this twig is for, and what it's going to be messing with. --4xx? What page would that be, anything in the nodes with an id of 400-499?

**Review by Mirzero — APPROVED** [review]
Looks like a much cleaner structure than we had.

File: src/web/themes/custom/workbc/templates/layout/page--4xx.html.twig (hunk 1 above)
Comment (zetagraph) [reply src/web/themes/custom/workbc/templates/layout/page--4xx.html.twig:1]: This is for the 404 pages. We would need to address a few things that don't fall under the node layout options.

**Pull request merged**